### ISSUE TITLE
wx._core.wxAssertionError in wx.Image after creating a PDFWindow 

### ISSUE THREAD

**Issue by david-hughes** [wxWidgets/Phoenix#1280]
Windows 10
wx version is 4.0.6 msw (phoenix) wxWidgets 3.0.5
Python is version 3.7.3 from python.org 

Creating a wx.image from a .png file works fine but when it follows the creation of a PDFWindow from wx.lib.pdfwin a wxAssertionError occurs that tries to blame the use of locale.setlocale() - not true either in the original app or in the following sample.

> wx._core.wxAssertionError: C++ assertion "strcmp(setlocale(LC_ALL, NULL), "C") == 0" failed at ..\..\src\common\intl.cpp(1579) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.
Things are going to break, please only change locale by creating wxLocale objects to avoid `this!`

```
import wx
from wx.lib.pdfwin import PDFWindow

app = wx.App(False)  # Create a new app, don't redirect stdout/stderr to a window.
frame = wx.Frame(None, wx.ID_ANY, "Hello World")
bitmap1 = wx.Image('reply4.png', wx.BITMAP_TYPE_PNG).ConvertToBitmap()
print(bitmap1)
pdfwin = PDFWindow(frame) 
bitmap2 = wx.Image('reply4.png', wx.BITMAP_TYPE_PNG).ConvertToBitmap()
print (bitmap2
frame.Show(True)     # Show the frame.
app.MainLoop()
```
![reply4](https://user-images.githubusercontent.com/2462978/60273652-7db28280-98ee-11e9-9de1-4362a8d3f40b.png)

**Comment by Metallicow** [comment]
Have you tried optimizing the image first...?
This may be the same issue popping up again... might look here #769

**Comment by RobinD42** [comment]
If the Acrobat activeX control or some other third party module changes the locale settings then it could trigger this error. There have been some recent reports  that creating and holding an instance of `wx.Locale(wx.LANGUAGE_DEFAULT)` after creating the app object will work around it.

**Comment by david-hughes** [comment]
Yes, creating and holding an instance of wx.Locale resolves the problem.

In fact I discovered I had already inserted the line:
`_locale2 = wx.Locale(wx.LANGUAGE_ENGLISH)  # suppresses wx assertion error`
into my application back in 2016 but the instance wasn't being held outside of app.OnInit()
At that time I didn't know it was the Acrobat activeX control being created that was the cause of the problem, so it required that to have happened **before** opening the totally unrelated dialog containing the bitmap. So testing my "solution" obviously didn't reproduce the circumstances of the original problem.
